CMakeLists.txt

```diff
@@ -138,6 +139,10 @@ if (BUILD_TESTS)
   add_subdirectory(tests)
 endif()
 
+if (BUILD_TUTORIALS)
+  add_subdirectory(tutorials)
+endif()
+
```

Comment:
Handle BUILD_TUTORIALS in https://github.com/Intel-Media-SDK/MediaSDK/blob/master/builder/profiles/fastboot.cmake, please